Libraries/pbxspec/Sources/Manager.cpp

```diff
@@ -86,9 +86,13 @@ findSpecification(std::vector<std::string> const &domains, std::string const &id
 {
     typename T::vector vector = findSpecifications <T> (domains, type);
```

Comment:
Yeah. That makes sense. In the meantime what do you recommend? Checking this in to at least fix the easy case of the bug? I'm not familiar with the `pbxbuild::DirectedGraph` code yet, but we should probably file an issue for that and take a look at pulling it out at some point.